### DISCUSSION TITLE
`npm run lint` does not work in a raw create-next-app project

### DISCUSSION THREAD

**Discussion by GregoryZeng** [vercel/next.js#61121] [Help]
### Summary

Hi,
I was trying to use eslint in my next.js project. However, I find that the `npm run lint` does not work in my create-next-app project. Here is the error message:
```
(base) ➜  test-eslint git:(main) npm run lint  

> test-eslint@0.1.0 lint
> next lint

Failed to load plugin '@typescript-eslint' declared in '../.eslintrc.js': Cannot find module '@typescript-eslint/eslint-plugin'
Require stack:
- /Users/zengsh/Projects/git/__placeholder__.js
Referenced from: /Users/zengsh/Projects/git/.eslintrc.js
```
The `test-eslint` is just a plain create-next-app project without any modification. The `npm run lint` should work as per the doc in https://nextjs.org/docs/app/building-your-application/configuring/eslint

Am I missing something? 

### Additional information

```js
(base) ➜  test-eslint git:(main) npx next info

Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:23 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6020
Binaries:
  Node: 18.17.0
  npm: 9.6.7
  Yarn: 3.6.1
  pnpm: 8.7.6
Relevant Packages:
  next: 14.1.0
  eslint-config-next: 14.1.0
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.3.3
Next.js Config:
  output: N/A


warn  - Latest canary version not detected, detected: "14.1.0", newest: "14.1.1-canary.9".
        Please try the latest canary version (`npm install next@canary`) to confirm the issue still exists before creating a new issue.
        Read more - https://nextjs.org/docs/messages/opening-an-issue
```
```


### Example

_No response_

**Comment by kimprosh** [comment]
Do you try: `npm i -D @typescript-eslint/eslint-plugin`?

**Reply by GregoryZeng** [reply]
I tried. Still not working.
```
(base) ➜  test-eslint git:(main) npm i -D @typescript-eslint/eslint-plugin

added 5 packages, and audited 366 packages in 3s

131 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
(base) ➜  test-eslint git:(main) ✗ npm run lint                             

> test-eslint@0.1.0 lint
> next lint

Failed to load plugin '@typescript-eslint' declared in '../.eslintrc.js': Cannot find module '@typescript-eslint/eslint-plugin'
Require stack:
- /Users/zengsh/Projects/git/__placeholder__.js
Referenced from: /Users/zengsh/Projects/git/.eslintrc.js
```

**Comment by icyJoseph** [comment]
Works for me, no issue. 

- What options did you use when creating the project?
- Maybe you have ESLINT installed globally? 
- What is `__placeholder__.js`?

**Reply by GregoryZeng** [reply]
Run `npx create-next-app@latest`. Enable typescript, eslint, tailwind css, src/ directory, app router. Disable import alias.

I check with `npm list -g eslint` with empty result.

Strangely, there is no such file as `__placeholder__.js` in the project root...

**Reply by icyJoseph** [reply]
What's up with the `git` directory? too? what the path going to the project?

**Reply by GregoryZeng** [reply]
the `git` directory is the parent of the project directory `test-eslint`. it is irrelevant to the nextjs (I have no idea why the error message relates to `/Users/zengsh/Projects/git/__placeholder__.js`...
